deploy/osd-dedicated-operator-mngmnt/03-dedicated-admins-manage-operators.ClusterRole.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
```

Why not use role aggregation?  Can remove both `SubjectPermissions` and put label on the ClusterRole.

```suggestion
metadata:
  labels:
    managed.openshift.io/aggregate-to-dedicated-admins: "project"
```